sample-apps/communication/main.js

```diff
@@ -73,9 +73,7 @@ async function onNearestPoiClick () {
       }]
     }
   }
-  // Geo-filter array: camX, camZ, max 2D horizontal distance (m)
-  // TODO(orenco): simplify API structure
-  const geofilter = [cameraCoords[0], cameraCoords[2], MAX_DISTANCE_H]
+  const geofilter = { camX: cameraCoords[0], camZ: cameraCoords[2], hDistance: MAX_DISTANCE_H }
```

Comment:
From @bergden-resonai :

I think camX and camZ are too specific, the terminology should be something like centerX centerZ to clarify you can also issue this search centered at any point in space - not necessarily around the user/camera

Reply:
Done